### DISCUSSION TITLE
How to get classes labels from MMPoseInferencer?

### DISCUSSION THREAD

**Discussion by volkov-maxim** [open-mmlab/mmpose#2302] [Q&A]
Hi, community!

I use MMPoseInferencer for inference on video and do it frame by frame.
I initialize the inferencer as next manner:

```python
inferencer = MMPoseInferencer(
    pose2d=pose_config,
    pose2d_weights=pose_checkpoint,
    det_model=det_config,
    det_weights=det_checkpoint,
    det_cat_ids=[0, 32],
    device=device,
)
```

As you can see I use the `det_cat_ids` parameter with a list of interested classes [0, 32] (human, sports ball). When I analyze the output of the inferencer I can't tell one class from another 🤷🏻‍♂️

**The question is**: How to get classes labels from MMPoseInferencer?

**Comment by NicolasPetermann134** [comment]
Could you answer that question?
